src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Dynamic Sampling
+sidebar_order: 10
+description: "Learn about the different methods for controlling the amount and type of data sampled for your project."
+---
+
+<Alert level="info">
+
+This content is provided on a working branch as part of a project that is in progress. We anticipate it will be changed as the project progresses.
+
+</Alert>
```

Comment:
```suggestion
<Note>

This content is provided on a working branch as part of a project that is in progress. We anticipate it will be changed as the project progresses.

</Note>
```